app-guides/minio.html.markerb

```diff
@@ -165,6 +170,151 @@ At this point `<NEW-USER>` can log in and read the contents of your buckets. Mak
 mc admin policy set miniotest readwrite user=<NEW-USER>
 ```
 
+## Multi-node deployment
+
+What if you want high availability? You can deploy MinIO in a multi-node configuration!
+
+Each node needs to know where to find the other nodes in the configuration. This can be done using either hostnames or IP addresses, but MinIO requires that they're sequential. We'll use hostnames since we can modify `/etc/hosts` to ensure their "sequential-ness". Ensuring sequential IP addresses is much more tricky!
+
+There's going to be a bit of back-and-forth here since we don't have any private IPs to put into `/etc/hosts` until we create the machines, but we need a Dockerfile to create them in the first place. Bear with me!
```

Comment:
```suggestion
There's going to be a bit of back-and-forth here since we don't have any private IPs to put into `/etc/hosts` until we create the Machines, but we need a Dockerfile to create them in the first place. Bear with me!
```